models/post-comments.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection.js");
+
+exports.fetchPostCommentByArticleId = (article_id, username, body) =>{
+
+    return db.query(`
+        SELECT * FROM articles
+        WHERE article_id = $1;`,[article_id])
+        .then((result) =>{
+            if (result.rows.length === 0){
+                return Promise.reject({status: 404, msg: "Article not found"});
+            };
+            return db.query(`
```

Comment:
I think because article_id is a foreign key you won't need to check the article exists as PSQL will throw an error if you try to add a comment to an article that already exists